packages/mdc-select/README.md

```diff
@@ -205,6 +205,20 @@ To disable a list item, set `aria-disabled` to `"true"`, and set `tabindex` to `
 </div>
 ```
 
+#### Resetting
```

Comment:
Thanks for submitting this PR. As of the most recent release (yesterday), you can reset the select by calling `setSelectedIndex(-1)` on the component. I overlooked adding that to the readme, but the process explained here is no longer required. 

We could change this to 
"Select elements can be reset, or cleared, by calling `setSelectedIndex(-1)`."